independent-projects/bootstrap/core/src/test/java/io/quarkus/bootstrap/resolver/maven/MavenRepoInitializerTest.java

```diff
@@ -0,0 +1,138 @@
+package io.quarkus.bootstrap.resolver.maven;

```

Comment:
Just to be consistent with the rest of the tests in this module, please use `io.quarkus.bootstrap.resolver.maven.test` as a package.